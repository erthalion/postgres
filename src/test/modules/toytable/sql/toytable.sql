CREATE EXTENSION toytable;

create table toytab (i int4, j int4, k int4) using toytable;

select * from toytab;

delete from toytab;

insert into toytab values(1, 2, 3);

update toytab set i = 4;

copy toytab from stdin;
