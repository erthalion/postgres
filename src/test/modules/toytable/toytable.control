comment = 'Dummy implementation of table AM api'
default_version = '1.0'
module_pathname = '$libdir/toytable'
relocatable = true
