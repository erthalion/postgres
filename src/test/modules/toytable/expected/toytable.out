CREATE EXTENSION toytable;
create table toytab (i int4, j int4, k int4) using toytable;
select * from toytab;
 i  | j  | k  
----+----+----
  1 |  1 |  1
  2 |  2 |  2
  3 |  3 |  3
  4 |  4 |  4
  5 |  5 |  5
  6 |  6 |  6
  7 |  7 |  7
  8 |  8 |  8
  9 |  9 |  9
 10 | 10 | 10
(10 rows)

delete from toytab;
ERROR:  Table access method doesn't support the operation
insert into toytab values(1, 2, 3);
ERROR:  Table access method doesn't support the operation
update toytab set i = 4;
ERROR:  Table access method doesn't support the operation
copy toytab from stdin;
ERROR:  Table access method doesn't support the operation
